docs/manual/common/concepts/ES_CQRS.md

```diff
@@ -1,59 +1,16 @@
-# Event Sourcing and CQRS
+# Managing data persistence
 
-Event sourcing and CQRS (Command Query Responsibility Segregation) are fundamental concepts behind Lagom's support for services that store information.
+As mentioned previously, each service should own its data and have direct access to the database. Other services must use the Service API to interact with the data. There must be no sharing of databases across different services since that would result in a too tight coupling between the services. Similar to a [Bounded Context](http://martinfowler.com/bliki/BoundedContext.html) in Domain-Driven Design terminology, each microservice should define a Bounded Context.
```

Comment:
This is a bad habit, thanks for calling me on it. It doesn't add value here, so I just removed it.